.github/workflows/deployment.yml

```diff
@@ -313,9 +314,9 @@ jobs:
           else
             release_args+=( --discussion-category "General" )
           fi
-          action="echo"
-          [ "$DO_PUBLISH" = "false" ] || action="command"
-          script/label-assets dist/gh_* | xargs $action gh release create "${release_args[@]}" -- 
+          guard="echo"
+          [ "$DO_PUBLISH" = "false" ] || guard=""
```

Comment:
This is using bash shortcircuiting logic to only set `guard=""` in the case that `$DO_PUBLISH` is not false right? i.e. if `$DO_PUBLISH` is anything other than `false`, we set `guard` to the empty string so that `gh release create` runs, rather than just `echo-ing` the command?

Reply:
Yes. If DO_PUBLISH is false, we want to print what would be done, but not actually cut a Release. It's a sort of a lightweight no-op mechanism.

Reply:
Oh gotcha